recipes/pastix/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "pastix" %}
+{% set version = "6.4.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: https://files.inria.fr/pastix/releases/v6/pastix-{{ version }}.tar.gz
+  sha256: 891d426188eed56c1075fb34d2d80132593a1536ffc05cf333567f68a4811e55
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('pastix', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ stdlib('c') }}                 # [unix]
+    - {{ compiler('c') }}               # [unix]
+    - {{ compiler('cxx') }}             # [unix]
+    - {{ compiler('fortran') }}         # [unix]
+    - {{ compiler('m2w64_c') }}         # [win]
+    - {{ compiler('m2w64_cxx') }}       # [win]
+    - {{ compiler('m2w64_fortran') }}   # [win]
+    - cmake
+    - ninja
+    - pkgconfig
+
+  host:
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
+    # - libscotch # no 64bit support
+    - libhwloc
+    - metis
+
+
+test:
+  commands:
+    # no idea how to test pastix...
```

Currently you're installing headers and binaries. You can check that those are present.
```suggestion
    # headers
    {% for each_lib in ["pastix", "spm"] %}
    test -f $PREFIX/include/{{ each_lib }}.h            # [unix]
    test -d $PREFIX/include/{{ each_lib }}              # [unix]
    if not exist %LIBRARY_INC%\{{ each_lib }}.h exit 1  # [win]
    if not exist %LIBRARY_INC%\{{ each_lib }} exit 1    # [win]
    {% endfor %}

    # shared libraries
    {% for each_lib in ["pastix", "pastixf", "pastix_kernels", "spm", "spmf"] %}
    test -f $PREFIX/lib/lib{{ each_lib }}.so                # [linux]
    test -f $PREFIX/lib/lib{{ each_lib }}.dylib             # [osx]
    if not exist %LIBRARY_BIN%\{{ each_lib }}.dll exit 1    # [win]
    {% endfor %}
```
You can also test for binaries, the existence of CMake/pkgconfig metadata, or even the correctness of that metadata, see e.g. the [`libgrpc`](https://github.com/conda-forge/grpc-cpp-feedstock/blob/main/recipe/meta.yaml) feedstock

Also, you're currently shipping all the executables under
```
share/doc/pastix/examples
share/doc/doc/examples
```
not sure if that is the intention...